console/src/label/Edit.tsx

```diff
@@ -126,7 +125,9 @@ export const Edit: Layout.Renderer = (): ReactElement => {
   });
 
   const arr = Form.useFieldArray<label.Label>({ path: "labels", ctx: methods });
-  const theme = Theming.use();
+  const theme = Layout.useSelectTheme();
+  const colors = theme?.colors.visualization.palettes.default ?? [];
```

Comment:
There's no reason this variable needs to be in the component body. Put it in the onClick handler of the button.